src/status_im2/contexts/communities/menus/community_options/view.cljs

```diff
@@ -6,8 +6,7 @@
 
 (defn hide-sheet-and-dispatch
   [event]
-  (rf/dispatch [:bottom-sheet/hide])
-  (rf/dispatch event))
+  (rf/dispatch [:bottom-sheet/hide-and-dispatch #(rf/dispatch event)]))
```

Comment:
what's the point of this change?

Reply:
I find this a bit odd. Can't we just make the bottom sheet events do the right thing entirely? Shouldn't they be responsible for also hiding any existing bottom sheet without the caller having to worry about this?